app/src/main/java/org/simple/clinic/editpatient/EditPatientUi.kt

```diff
@@ -0,0 +1,25 @@
+package org.simple.clinic.editpatient
+
+import org.simple.clinic.editpatient_old.PatientEditValidationError
+import org.simple.clinic.patient.Gender
+import org.simple.clinic.widgets.ageanddateofbirth.DateOfBirthAndAgeVisibility
+import org.threeten.bp.LocalDate
+
+interface EditPatientUi {
```

Comment:
Shouldn't this be in the `_old` package?

Reply:
I'm reusing classes when possible.